decision-log/readme.md

```diff
@@ -0,0 +1,79 @@
+# Design decision log
+
+Not all requirements can be captured in the beginning of an engagement during an architecture design session or even after having written the Game Plan document. The initial architecture design can evolve or change during an engagement, especially if there are multiple possible technology choices that can be made. Tracking these changes within a large document is in most cases not ideal, as one can lose oversight over the design changes made at which point in time. Having to scan through a large document to find a specific content takes time, and in many cases the consequences of a decision is not documented.
+
+## Why is it important to track design decisions
+
+Tracking an architecture design decision in a log can have many advantages:
+
+- Developers and project stakeholders can see the decision log and track the changes, even as the team composition changes over time.
+- The log is kept up-to-date.
+- The context of a decision including the consequeces for the team are documented with the decision.
+- It is easier to find the design decision in a log than having to read a large document.
+
+## What is a great and recommended format for tracking decisions
+
+In his blog, Michael Nygard proposed the use of an [Architecture Decision Record](http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions).
+
+### Architecture Decision Record (ADR)
```

Comment:
Some decisions are made quickly without having a detailed study of the pros and cons of the options, and some are more elaborate and it would be good to document these. I'll add a reference to the design doc and also to the Trade studies. Thanks for your suggestions @charleszipp!